src/CoreBundle/private/library/classes/TFullGroupTable/TFullGroupTable.class.php

```diff
@@ -878,11 +891,16 @@ protected function _BuildFilterSection()
                 TGlobal::OutHTML($this->onClick)
             );
 
+            // We need the key event; that is not given when simply specifying "onkeypress" in the input:
+            $filterContent .= '<script>
+                    document.querySelector("#searchLookup").addEventListener("keypress", resetPagingAndSearch);
```

Comment:
Do we have cases, where multiple tables are on the page at the same time? If yes, then this would only attach a listener to the first one. In fact, if there can be multiple lists on a page, then the definition of `resetPagingAndSearch` will also be duplicated - in that case that should also be moved into `document.{LIST_NAME}`.

Reply:
Also: The `keypress` event seems to be deprecated. `input` seems to be the preferred event nowadays.

Reply:
No multiple tables.

Changed to input (event).